test/optimizer/JSDCE-objectPattern-output.js

```diff
@@ -0,0 +1,9 @@
+let d = 40;
+
+let z = 50;
+
+globalThis.f = function(r) {
+ let {a: a, b: b} = r;
```

Comment:
Why does the output not use the shorthand notation here ?  i.e. can't this just be `{a, b}`?

Reply:
That's how `terserAst.print_to_string` emits it. Acorn parses it internally in a uniform manner, so this is a matter of the AST-to-string process not special-casing the short form. There might be an optimization that does that, but in general we don't run the terser optimizer (we rely on closure to do this sort of thing).

Reply:
I verified there is no format option (like there is an option for whitespace or not) for this, so if terser can do it (which I'd bet it can) it's an optimization, which we don't run.

Reply:
Seems odd to regress code size here, but I guess closure can erase it.